AWSSDKSwiftCLI/Package.resolved

```diff
@@ -0,0 +1,95 @@
+{
+  "pins" : [
```

Comment:
This file is committed so that when we run the command line tools, we all get the same versions of dependencies until we choose to update them.